persistent-postgresql/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for persistent-postgresql
 
+## 2.12.0.1
```

Comment:
does this merit a more significant bump than like, a patch?  I'm not sure what the versioning convention for this project is.

Reply:
An addition to the API is a minor version bump - so since we're adding a function, we'd go to 2.12.1.0